HW11/helicopter.py

```diff
@@ -0,0 +1,43 @@
+from HW11.vehicle import Vehicle
+
+
+# Define the Helicopter class, which inherits from the Vehicle class
+# Inheritance
+
+class Helicopter(Vehicle):
+
+    def recharge(self):
+        pass
+
+    def __init__(self, max_speed, rotor_diameter, year, model):
+        super().__init__(max_speed)
+        self.rotor_diameter = rotor_diameter
+        self.year = year  # public attribute
+        self.__model = model  # private attribute
+
+    def __get_model(self):  # Private method
+        return self.__model
+
+    def get_model(self):  # public method which use private method
+        return self.__get_model()
+
+    def get_max_speed(self):
+        return self.max_speed
+
+    def set_max_speed(self, new_max_speed):
+        self.max_speed = new_max_speed
```

Comment:
in python we use @property not getters_setters

Reply:
I thinked - all methods must be used in homework with different samples, even getters, setters :(